examples/terraform-aws-ecs-example/main.tf

```diff
@@ -2,6 +2,9 @@
 # PIN TERRAFORM VERSION TO >= 0.12
 # The examples have been upgraded to 0.12 syntax
 # ---------------------------------------------------------------------------------------------------------------------
+provider "aws" {
+  region = "us-east-1"
```

Comment:
Did we want to make this a var instead of hardcoded string?

Reply:
Yes - thanks!

Reply:
Fixed in f5c6a3f